src/api/firebase.js

```diff
@@ -66,7 +72,12 @@ export async function addItem(listId, { itemName, daysUntilNextPurchase }) {
 	});
 }
 
-export async function updateItem() {
+export async function updateItem(listId, itemId, { day, purchaseCounter }) {
+	const listCollectionRef = doc(db, listId, itemId);
```

Comment:
A general suggestion for async functions is to wrap them in a try/catch block for better error handling. 😊

Reply:
Thanks. try/catch block has been added